packages/logger/src/Logger.ts

```diff
@@ -311,15 +309,6 @@ class Logger extends Utility implements ClassThatLogs {
     return this.logEvent;
   }
 
-  /**
-   * It returns a boolean value, if true all the logs will be printed.
-   *
-   * @returns {boolean}
-   */
-  public getLogsSampled(): boolean {
-    return this.logsSampled;
-  }
-
```

Comment:
Putting this comment here because the line is not part of the diff, but I would modify also the `createChild` method to make it more explicit that the child is getting the `sampleRateValue` from the parent, to this:

```ts
const parentsOptions: ConstructorOptions = {
  logLevel: this.getLevelName(),
  customConfigService: this.getCustomConfigService(),
  logFormatter: this.getLogFormatter(),
  sampleRateValue: this.powertoolLogData.sampleRateValue,
};
```